src/index.tsx

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react-refresh/only-export-components */
```

Comment:
Oo I've been meaning to do this. Thanks!

Reply:
I think we probably shouldn't disable it in many places, but for index files I think it makes sense!